classes/Database.php

```diff
@@ -109,6 +109,29 @@ public function all_cards($max_version = '99.99.99', $lang = 'en')
         return $result;
     }
 
+    public function all_versionsOfCard($code, $lang = 'en') {
```

Comment:
Maybe return the complete cards from these functions? The added flexibility might come in handy at some point, the current use case needs to map anyway to build the options, and it's not like we need to keep the data minimal when it's all used server-side. Could also add API endpoints for these.

Reply:
Return values changed, but API endpoints will only happen once something actually needs them - they're pretty quick to make and pretty unrelated to the topic of this PR.